CMakeLists.txt

```diff
@@ -80,6 +80,7 @@ if(NOT CMAKE_BUILD_TYPE)
         set(CMAKE_BUILD_TYPE "Release")
 endif()
 
+# XXX: ASAN is currently broken
```

Comment:
Please, change the three initial letter for another group that can have the same meaning.